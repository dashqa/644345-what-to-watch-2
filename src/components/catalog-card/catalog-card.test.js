import React from 'react';
import renderer from 'react-test-renderer';
import CatalogCard from './catalog-card';

it(`Catalog card correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<CatalogCard
      title={`Spider Man`}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
