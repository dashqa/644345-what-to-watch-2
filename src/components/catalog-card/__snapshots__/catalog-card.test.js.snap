// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog card correctly renders after relaunch 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt=""
      height="175"
      src=""
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/0"
      onClick={[Function]}
    >
      
    </a>
  </h3>
</article>
`;
